dwm1001/src/lib.rs

```diff
@@ -798,14 +803,14 @@ impl Led {
     pub fn enable(&mut self) {
         // https://github.com/braun-robotics/rust-dwm1001/issues/94
         #[allow(deprecated)]
-        self.0.set_low()
```

Comment:
You should be able to remove the deprecated allows now, since you are no longer using the deprecated API.